contrib/kube-prometheus/hack/grafana-dashboards-configmap-generator/bin/grafana_dashboards_generate.sh

```diff
@@ -0,0 +1,317 @@
+#!/bin/bash
```

Comment:
please add this lines on the header of these shell scripts so it always fail fast

```
# exit immediately when a command fails
set -e
# only exit with zero if all commands of the pipeline exit successfully
set -o pipefail
# error on unset variables
set -u
```

Reply:
@gianrubio : changes done and script tested. One quick comment... after adding set -e I had to add `|| true` in this type of commands:
```
    (( total_files_processed++ )) || true
```
please take a look to the code now and let me know your thoughts.